File: e2e/src/test/resources/projects/monolith-clinical-project/src/main/java/com/example/clinical/config/DatabaseConfiguration.java
package com.example.clinical.config;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@Configuration
@EnableJpaRepositories("com.example.clinical")
@EntityScan("com.example.clinical")
@EnableJpaAuditing(auditorAwareRef = "springSecurityAuditorAware")
@EnableTransactionManagement
public class DatabaseConfiguration {

}

File: e2e/src/test/resources/projects/order-fulfillment-aggregates/src/main/kotlin/io/zenwave360/example/config/DatabaseConfiguration.kt
package io.zenwave360.example.config

import org.springframework.boot.persistence.autoconfigure.EntityScan
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.transaction.annotation.EnableTransactionManagement

@Configuration
@EnableJpaRepositories("io.zenwave360.example.orderfulfillment")
@EntityScan("io.zenwave360.example.orderfulfillment")
@EnableJpaAuditing(auditorAwareRef = "springSecurityAuditorAware")
@EnableTransactionManagement
open class DatabaseConfiguration
